config.json

```diff
@@ -16,7 +16,7 @@
     "DO_NOT_BUILD": [
       "dkp/kommander/2.3/**",
       "dkp/konvoy/2.3/**",
-      "dkp/kaptain/2.0.0/**"
+      "dkp/kaptain/2.1/**"
```

Comment:
note to self: this will end up conflicting with #4376

Reply:
I think there's another place we need to change @juliangieseke

Reply:
https://github.com/mesosphere/dcos-docs-site/blob/jg/add-kaptain-to-dkp-2-rn/s3config.json#L13

Reply:
good catch, thanks! updated 👍